tosfs/core.py

```diff
@@ -413,6 +416,200 @@ def _try_dir_info(self, bucket: str, key: str, path: str, fullpath: str) -> dict
         except Exception as e:
             raise TosfsError(f"Tosfs failed with unknown error: {e}") from e
 
+    def exists(self, path: str, **kwargs: Union[str, bool, float, None]) -> bool:
+        """Check if a path exists in the TOS file system.
+
+        Parameters
+        ----------
+        path : str
+            The path to check for existence.
+        **kwargs : dict, optional
+            Additional arguments.
```

Comment:
The pydoc says it will be a dict, while the declared arg is a `Union` ?  Is this correct ?